qi-doc/scribblings/list-operations.scrbl

```diff
@@ -54,23 +54,65 @@ Deforestable version of @racket[take] from @racketmodname[racket/list].
 
 @section{Consumers}
 
-foldr
+@defproc[(foldl (proc procedure?) (init any/c) (lst list?) ...+) any/c]{
```

Comment:
These changes from 2251e5906a84563b3e585b5e6118d7cf4881b41d belong with the previous commit.